src/main/java/org/assertj/core/api/AbstractLocalDateTimeAssert.java

```diff
@@ -317,6 +321,30 @@ public SELF isNotIn(String... dateTimesAsString) {
     return isNotIn(convertToLocalDateTimeArray(dateTimesAsString));
   }
 
+  /**
+   * Verifies that the actual {@link LocalDateTime} is close to the current date and time on the UTC timezone,
+   * according to the given {@link TemporalUnitOffset}.
+   * You can build the offset parameter using {@link Assertions#within(long, TemporalUnit)} or {@link Assertions#byLessThan(long, TemporalUnit)}.
+   * <p>
+   * Example:
+   * <pre><code class='java'> LocalDateTime actual = LocalDateTime.now(Clock.systemUTC());
+   *
+   * // assertion will pass
```

Comment:
to be explicit let's add: `as it is executed less than one second after actual was built`